packages/glsp-playwright-core/src/remote/locator.ts

```diff
@@ -0,0 +1,22 @@
+import type { Locator, Page } from '@playwright/test';
+import type { GLSPApp } from '~/glsp/app';
+
+export class GLSPLocator {
+    constructor(protected readonly locator: Locator, public readonly app: GLSPApp, public readonly parent?: GLSPLocator) {}
+
+    get page(): Page {
+        return this.app.page;
+    }
+
+    locate(): Locator {
```

Comment:
I think `locator` would be a better name here since this is what it returns, the playwright Locator. `locate` somehow sounds to me like a position or identifier should be returned. What do you think?

Reply:
I used `locate` because there is a context switch from the `GLSPLocator` to the Playwright `Locator`. Further, we are using the `GLSPLocator` everywhere and it is also always called `locator`. In this case, the developer would read `locator.locator` if they wanted to switch to the `Playwright` context. We can rename it to `playwright` or `playwrightLocator` or something similar.